ios/keyman/Keyman/Keyman/AppDelegate.swift

```diff
@@ -91,6 +113,74 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     return _overlayWindow!
   }
 
+  public func installAdhocKeyboard(url: URL) {
+    let documentsDirectory = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask)[0]
```

Comment:
Is there `packages/` in this documentsDirectory to contain all the installed keyboard packages? e.g.
```
documents/packages/gff_amh_7/
documents/packages/khmer10/
```

Reply:
So all of the keyboards and assets are installed a shared storage directory so the keyboard extension can use it.  The documents directory is only used for unzipping, parsing, and installing.